samples/peripheral/radio_test/src/radio_cmd.c

```diff
@@ -14,7 +14,8 @@
 
 #include "radio_test.h"
 
-#define USE_MORE_NRF52_RADIO_POWER_OPTIONS (defined(NRF52840_XXAA))
+#define USE_MORE_NRF52_RADIO_POWER_OPTIONS (defined(NRF52840_XXAA) || defined(NRF52820_XXAA) || \
```

Comment:
Isn't there a define that could be used instead of maintaining this list?

Reply:
I would suggest to replace the use of this define with set of #if defined(RADIO_TXPOWER_TXPOWER_Pos8dBm) per TX power definition.

Reply:
Could be, probably this list is here because this radio definition are used in few places in this sample so there will be a lot of #if defined(RADIO_TXPOWER_TXPOWER_xxxxxxxxxx) sentence. But it isn't a problem for me to remove this list and use other defines set.

Reply:
I think that would be more future-proof. At the moment this list needs to be updated in case of new chips that support it. Instead of relying on the MDK defines.
Also other macros like this should be removed in favour of MDK defines, like USE_MORE_RADIO_MODES

Reply:
Ok. Done